app/src/shared/components/ActivityList/ActivityListItem/index.jsx

```diff
@@ -0,0 +1,229 @@
+import routes from '$routes'
+import usePending from '$shared/hooks/usePending'
+import { productTypes } from '$mp/utils/constants'
+
+const Container = styled.div`
+    display: grid;
+    grid-template-columns: 32px auto;
+    grid-column-gap: 16px;
+    width: 100%;
+    border-bottom: 1px solid #f5f5f5;
+    padding: 16px;
+
+    &:last-child {
+        border-bottom: none;
+    }
+
+    &:hover {
+        background-color: #f8f8f8;
+    }
+`
+
+const ImageContainer = styled.div`
+    max-height: 32px;
+`
+
+const StyledSpinner = styled(Spinner)`
+    position: relative;
+    top: -24px;
+    left: 8px;
+`
+
+const StyledAvatar = styled(Avatar)`
+    height: 32px;
+    border-radius: ${(props) => (props.circle ? '50%' : '4px')};
+    overflow: hidden;
+    filter: ${(props) => (props.isLoading ? 'brightness(70%)' : 'none')};
+`
+
+const TextContent = styled.div`
+    display: grid;
+    grid-template-rows: auto 18px;
+    font-size: 14px;
+    line-height: 18px;
+`
+
+const Text = styled.div`
+    min-height: 18px;
+    max-height: 36px;
+    overflow: hidden;
+    color: #323232;
+
+    // && is used to increase specificity so we can break free from parent
+    // styles without using !important
+    // https://styled-components.com/docs/faqs#how-can-i-override-styles-with-higher-specificity
+    && a {
+        display: inline-block;
+        padding: 0;
+        background-color: transparent;
+        font-weight: 500;
+        line-height: 18px;
+    }
+
+    ${Container}:hover && a {
+        color: #0324FF;
+    }
+`
+
+const Details = styled.div`
+    color: #a3a3a3;
+    font-size: 12px;
+`
+
+const Type = styled.span`
+    color: #323232;
+`
+
+type Props = {
+    activity: Activity,
+    user?: any,
+    resource?: any, // Product | Canvas | Stream
+    resourceType?: ?ResourceType,
+}
+
+const renderItem = (action: string, linkTitle: ?string, linkHref: ?string, id: ?string) => (
+    <React.Fragment>
+        {linkHref != null ? (
+            <a
+                href={linkHref}
+            >
+                {linkTitle}
+            </a>
+        ) : (
+            id
+        )}
+        &nbsp;
+        {I18n.t(`shared.action.${action.toLowerCase()}`)}
+    </React.Fragment>
+)
+
+const renderStreamItem = (id, stream, action) => (
+    renderItem(
+        action,
+        stream && stream.name,
+        stream && routes.stream({
+            id: stream.id,
+        }),
+        id,
+    )
+)
+
+const renderProductItem = (id, product, action) => (
+    renderItem(
+        action,
+        product && product.name,
+        product && routes.editProduct({
+            id: product.id,
+        }),
+        id,
+    )
+)
+
+const renderCanvasItem = (id, canvas, action) => (
+    renderItem(
+        action,
+        canvas && canvas.name,
+        canvas && routes.canvasEdit({
+            id: canvas.id,
+        }),
+        id,
+    )
+)
+
+const renderContent = (activity, resource, resourceType) => {
```

Comment:
Missed Dashboards?

Reply:
They are left out intentionally since they are kind of a dying breed